Uncheck scc beta filter and assert screen change

QingmingSu opened the pull request:
Make sure 'Hide Beta Versions' unchecked before going forward,
otherwise it will cause problem such as the cursor is back to
scc-beta-filter-checkbox area unexpectly

- Related ticket: https://progress.opensuse.org/issues/34465
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/823
- Verification run: http://openqa-apac1.suse.de/tests/822#step/patch_before_migration/30

okurz commented:
Yes, the `wait_screen_change` can help but I think a proper needle check for the unchecked "hide beta version" would be a better approach: More explicit, saver as we wait for the real expected result and can be even faster so we would save time. Please use a proper assert_screen

QingmingSu commented:
Sure, but new needle will be required then.

okurz commented:
yes, of course. I do not see this as a problem. You can even create the needle on the production instance already

QingmingSu commented:
Changed to assert screen with new needle "scc-beta-filter-unchecked".
Also updated PR subject and description.

Note that it is not required for SLE 15 installation, the issue is already prevented by ```assert_screen($modules_needle);```

QingmingSu commented:
Yes, a new needle is required, I just created textmode needle with tag scc-beta-filter-unchecked for sle 12-sp3. So, your test (< sle 15) will fail if there is scc registration not in textmode. Sorry for any inconvenience caused.
